app/styles/profile.css

```diff
@@ -1,58 +1,114 @@
-.profile-background {
-  background: var(--nav--bg);
-  height: 50vh;
+.profile-container {
+  display: grid;
+  grid-template-columns: repeat(12, minmax(0, 1fr));
+  gap: 2rem;
+  max-width: min(90%, 1536px);
+  margin: 3rem auto 0 auto;
 }
+
 .profile-form {
-  margin-top: calc(-45vh);
-  background-color: var(--body-bg-color);
-  box-shadow: 0px 4px 4px 2px var(--info-tab--before--box-shadow);
-  border-radius: 10px;
-  max-height: 75vh;
-  min-width: 350px;
-  max-width: 600px;
-  margin-bottom: 0;
+  grid-column: 5 / 9;
 }
+
 .profile-form-grid {
   display: grid;
-  grid-template-columns: 1fr 1fr;
-  grid-gap: 16px;
+  grid-template-columns: 1fr;
+  gap: 1rem;
   margin-top: 2rem;
 }
-.profile-field p {
-  border-top: 2px solid var(--not-assign--text);
+
+.profile-field {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+  position: relative;
+  padding-bottom: 2rem;
+}
+
+.profile-field-input-container {
+  display: grid;
+  grid-template-columns: auto 1fr auto;
+  align-items: center;
+  border-radius: 8px;
+  border: 1px solid #d1d5db;
+  background-color: #f9fafb;
```

Comment:
Can we please CSS variables for the color, there should be a file present which contains all the variables ?